src/middlewared/middlewared/plugins/jail.py

```diff
@@ -571,13 +571,26 @@ def list_resource(self, resource, remote, want_cache, branch):
                     # iocage returns - for None
                     plugin[i] = elem if elem != "-" else None
 
+                # plugin[1] is the UUID
```

Comment:
Also i believe `0` index contains the plugin name, not sure but just did `iocage.fetch` locally and it seems the plugins description is at `1`.

Reply:
iocage.fetch isn't the same, this iterates on all the elements. 1 is the name :)